FRW/Services/Manager/Config/RemoteConfig.swift

```diff
@@ -202,6 +203,9 @@ extension RemoteConfigManager {
                 }
             case .isIOS:
                 return true
+            case .insufficientStorage:
```

Comment:
Let's make this func `boolValue` async, otherwise, the user  might missed their alert on home page.

Reply:
That could potentially delay the UI, and I wouldn't feel comfortable making an enum method async, as it would imply adding logic that should reside elsewhere.
For that reason I opted for using the event bus, so that notifications are reloaded when the account info has been retrieved.